Makefile

```diff
@@ -522,7 +522,8 @@ build-helm-tests: operations/helm/charts/mimir-distributed/charts
 	@./operations/helm/tests/build.sh
 
 check-helm-tests: build-helm-tests
-	@git diff --exit-code -- ./operations/helm/tests || (echo "Please rebuild helm tests output 'make build-helm-tests'" && false)
+	@git diff --exit-code -- ./operations/helm/tests || (echo "Difference found. Please rebuild helm tests output 'make build-helm-tests'" && false)
+	@git ls-files --others --exclude-standard | grep "^operations/helm/tests" && (echo "Untracked files found. Please rebuild helm tests output 'make build-helm-tests'" && false) || true
```

Comment:
We do this also at least with jsonnet. WDYT about a script to do these checks that we can call from the makefile? It should take a path and an error message? Or a make target.

Reply:
actually diff is used in 10 places, most of them could use this fix I think

Reply:
this turned out to be a bit more complicated than I expected with having to support multiple paths and the fact that we need to invert the grep result code